packages/web-components/src/components/leadspace/leadspace.ts

```diff
@@ -164,13 +171,13 @@ class C4DLeadSpace extends StableSelectorMixin(LitElement) {
     const { gradientStyleScheme, type, size } = this;
     return html`
       <section class="${this._getTypeClass()}" part="section">
-        <div class="${c4dPrefix}--leadspace__container">
-          <div class="${this._getGradientClass()}">
+        <div class="${c4dPrefix}--leadspace__container" part="container">
+          <div class="${this._getGradientClass()}" part="gradient">
```

Comment:
This is more accurately `overlay`, then we can use just `gradient` for the actual gradient.
```suggestion
          <div class="${this._getGradientClass()}" part="overlay">
```